cclib/parser/orcaparser.py

```diff
@@ -616,6 +618,27 @@ def splitter(line):
             line = next(inputfile)
             self.metadata["t1_diagnostic"] = utils.float(line.split()[-1])
 
+        # SCF energy with basis cc-pVDZ:                          -40.197189724
+        # SCF energy with basis cc-pVTZ:                          -40.212905351
+        # Extrapolated CBS SCF energy (2/3) :                     -40.218016328 (-0.005110977) 
+
+        # MDCI energy with basis cc-pVDZ:                          -0.186922179
+        # MDCI energy with basis cc-pVTZ:                          -0.224166487
+        # Extrapolated CBS correlation energy (2/3) :              -0.245929689 (-0.021763202)
+
+        # Estimated CBS total energy (2/3) :                      -40.463946018
+
+
+
+        # -------------------------   --------------------
+        # FINAL SINGLE POINT ENERGY       -40.463946017697
+        # -------------------------   --------------------
+        if line[:25] == 'FINAL SINGLE POINT ENERGY':
```

Comment:
did you talk to the other devs about this property?

Reply:
We did talk about it, but I can't remember the entire conclusion. Do you expect it to be present for NWChem, Gaussian, ... and not just ORCA? Based upon your other proposals, you assume this *isn't* the G4 (or some other thermochemical) energy, right?

The risk to me is that
> is not directly parsable from most programs

means we would have to construct it manually. That might be ok depending on what components you expect to be present.

Reply:
I want to retrieve this property for all the programs. The goal is to have a unified parameter that could be used as a single-point energy correction (SPC). In ORCA, "FINAL SINGLE POINT ENERGY" contains the final SPC that you want to use. In Gaussian, the final SPC value that you want to use can be specified in different ways, but we could adapt the code that GoodVibes uses to detect these energies.

The problem I had when trying to adapt the energy values from cclib with GoodVibes/AQME is that QM programs sometime give different types of energy values that get parsed, so I couldn't use a single parameter to choose the final energy that I wanted to use as SPC.

An example in Gaussian for G4 calcs, you get multiple energy values during the same calculation, but the final energy you want to use for a SPC is that of the G4 protocol. However, in other calculations, the energy values you want to use are those coming from DFT or TDDFT calcs, which are contained in different properties of the cclib object or JSON.

Same for ORCA, for example when using CASSCF-NEVPT2 calculations, you get different types of energy, but for SPC you only want the final NEVPT2 energy. When you change the calc type (for example, to DLPNO energy calcs), the value of the SPC isn't NEVPT2 anymore (i.e. the DLPNO energy in this case). 

Let me know if you need further information on this issue or the explanation wasn't clear enough!

Reply:
Hi folks, excuse me for jumping in. @jvalegre, am I following you correctly in that what you are looking for is a single property that represents the 'final' energy of the calculation? ie, at the highest level of theory available?
So for for standard HF/DFT, this would be SCF, for MP2 (where HF and MP2 are available) this would be MP2, for MP3 (where HF, MP2 and MP3 is available) this would be MP3 and so on?

If so, I can see this being a useful property, particularly in more complicated calculations where the 'correct' choice may not be intuitive to the user. Whether this should be a numerical property or a pointer to one of the other properties though I'm less sure. Ideally whatever this final energy is, it should already have been parsed into another property, and at the very least it would be useful to indicate _which_ energy is stored there.

Reply:
Hi @oliver-s-lee  - that's correct, I want to retrieve the energy from the highest level of theory available. In my case, it's useful for DLPNO single-point energy corrections. I'm attaching an example file, in which the final energy should be:

```
-------------------------   --------------------
FINAL SINGLE POINT ENERGY     -1365.394243212048
-------------------------   --------------------
```

[IntI-CF2-CF2H,OH,ax-a_DLPNO.txt](https://github.com/cclib/cclib/files/12484217/IntI-CF2-CF2H.OH.ax-a_DLPNO.txt)